scss/typography/_base.scss

```diff
@@ -506,4 +506,9 @@ $abbr-underline: 1px dotted $black !default;
       border-radius: $keystroke-radius;
     }
   }
+
+  // Use to create an initial aka dropcap
+  .dropcaps {
+    @include dropcaps();
+  }
```

Comment:
Sorry, I misnamed the file for the default styles `.dropcaps`. I meant `scss/typography/_helpers.scss`.
Also, please move it to `scss/typography/_dropcaps.scss` to its own mixin for default styles `@mixin foundation-dropcaps`.

Then, you can include `@mixin foundation-dropcaps` in `@mixin foundation-typography`.